packages/comet-utils/src/quantification.ts

```diff
@@ -406,9 +406,11 @@ const createQuantificationSummary = async (
         tenYearProjectedBaselineTonnesPerYear
       : tenYearProjectedFutureTonnesPerYear;
   const totalAcres = totalM2 * 0.000247105;
-  const grandfatheredYears = CURRENT_YEAR - Number(grandfatherableYears[0]);
+  const firstGrandfatherableYear =
+    Number(grandfatherableYears[0]) || CURRENT_YEAR;
+  const grandfatheredYears = CURRENT_YEAR - firstGrandfatherableYear;
 
-  const switchYear = grandfatherableYears[0];
+  const switchYear = firstGrandfatherableYear.toString();
```

Comment:
Possible at all to change years to number to match the type of years elsewhere?

Reply:
yeah, but we should probably migrate all quantification entities first.